front/src/applications/editor/components/InfraErrors/InfraErrorCorrector.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { getEditorState } from 'reducers/editor/selectors';
+import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
+import magicWandIcon from 'assets/pictures/magic_wand.svg';
+import InfraErrorCorrectorModal from './InfraErrorCorrectorModal';
+
+const InfraErrorCorrector: FC = () => {
```

Comment:
Don't think the FC is required here without any props or childrens
```suggestion
const InfraErrorCorrector = () => {
```